### PR TITLE
Create OP batch inbox transactions module

### PR DESCRIPTION
**Pull request by catalyst17**
Substreams module to track all the transactions to the OP Batch Index EOA and extract its calldata

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 -include .env
 export
 
-START_BLOCK ?= 4051031
-STOP_BLOCK ?= 0
+START_BLOCK ?= 19028181
+STOP_BLOCK ?= 19028182
```
Comment (iuwqyir) [inline Makefile:5]: Are these just for development?

**Review by iuwqyir — APPROVED** [review]
Do we choose what we want here from the transaction or is this later on in the pipeline?
Didn't see the place

File: Makefile (hunk 1 above)
Comment (catalyst17) [reply Makefile:5]: yes it's just for verifying the data is properly indexed in the example blocks

**Comment by catalyst17** [comment]
> Do we choose what we want here from the transaction or is this later on in the pipeline? Didn't see the place

since I didn't have the full picture of what data will we get after decoding later on in the pipeline and what data should be added by the substreams module itself I left it like this

but now it seems that some parts like `batcher address` and `inbox address` should be added here, I'll do it

**Pull request merged**